lblod/frontend-gelinkt-notuleren
# GN-5233: mandataris-rank - take diaeresis into account

### Overview
This PR ensures that letters with a diaeresis are taken into account when converting mandatee ranks to numbers.

##### connected issues and PRs:
[GN-5233](https://binnenland.atlassian.net/browse/GN-5233?atlOrigin=eyJpIjoiNTVlMTQ4MWI5MWY3NGZiNzgyZTUwZTJhNGE4ZTU2NjIiLCJwIjoiaiJ9)

### How to test/reproduce
- Open the frontend
- Create/open an IV
- Ensure the 'IVGR4-LMB-1-rangorde-gemeenteraadsleden' is configured somewhere; and the source data (gemeenteraad) should contain at least 24 mandatees
- Sync the meeting
- Ensure the mandatees are correctly ordered based on rank (including 'TWEEËNTWINTIGSTE' and 'DRIEËNTWINTIGSTE')

### Checks PR readiness
